pkg/terraform/aws/terraform_aws.go

```diff
@@ -210,16 +212,18 @@ func SetOutput(rootBody *hclwrite.Body, useEIP bool) {
 // RunTerraform executes terraform apply function that creates the EC2 instances based on the .tf file provided
 // returns a list of AWS node-IDs and node IPs
 func RunTerraform(terraformDir string, useEIP bool) ([]string, []string, error) {
+	var stdBuffer bytes.Buffer
+	var stderr bytes.Buffer
+	mw := io.MultiWriter(os.Stdout, &stdBuffer)
```

Comment:
may you try to use SetupRealtimeCLIOutput ?

Reply:
I would leave it unchanged as I don't think this PR touches this area